src/migrations/20240503093317_alter_table_add_isdelete_totickets.ts

```diff
@@ -6,4 +6,8 @@ export async function up(knex: Knex): Promise<void> {
   });
 }
 
-export async function down(knex: Knex): Promise<void> {}
+export async function down(knex: Knex): Promise<void> {
+  return knex.schema.alterTable("tickets", (table) => {
+    table.dropColumn("isDelete");
```

Comment:
Inconsistent column naming, why mixes both `snake_case` and `camelCase` convention?